lib/ticket-types.js

```diff
@@ -2,7 +2,7 @@
 
 function ticketTypes (args, callback) {
   var ticketTypes = [
-    {name: 'ninja', title: 'Ninja', tooltip: 'Select Ninja to create tickets for attendees under 13 or over 13.'},
+    {name: 'ninja', title: 'Youth', tooltip: 'Select Youth to create tickets for attendees under 18.'},
```

Comment:
not a big deal in any case; just wondering if in the future this could cause confusion to someone reading the source. Maybe a comment could help in that situation.